template-parts/content/entry_thumbnail.php

```diff
@@ -24,7 +24,7 @@
 if ( is_singular( get_post_type() ) ) {
 	?>
 	<div class="post-thumbnail">
-		<?php the_post_thumbnail( 'full', [ 'class' => 'skip-lazy' ] ); ?>
+		<?php the_post_thumbnail( 'wp-rig-post-thumbnail', [ 'class' => 'skip-lazy' ] ); ?>
```

Comment:
Needs to be "wp-rig-featured", not "wp-rig-post-thumbnail".

Reply:
Fixed. Thank you for having noticed this mistake.